core/src/main/kotlin/org/evomaster/core/problem/rest/classifier/GaussianOnlineClassifier.kt

```diff
@@ -36,18 +36,37 @@ class GaussianOnlineClassifier : AIModel {
     private var density200: Density? = null
     private var density400: Density? = null
 
+    // classifier state
+    private var cp: Int = 0        // correct predictions
+    private var tot: Int = 1       // total requests
+    private var ac: Double = 0.0   // accuracy ratio
+
     fun setDimension(d: Int) {
         require(d > 0) { "Dimension must be positive." }
         this.dimension = d
         this.density200 = Density(d)
         this.density400 = Density(d)
+        this.cp = 0
+        this.tot = 1
+        this.ac = 0.0
     }
 
     fun getDimension(): Int {
         check(this.dimension != null) { "Classifier not initialized. Call setDimension first." }
         return this.dimension!!
     }
 
+    fun setCp(cp: Int) {
```

Comment:
these setters and getters are redundant in Kotlin, as automatically generated by compiler if you declare it like `var correctPredictions: Int = 0` (note the not use of `private` here, as it seems you need get and set both to be public).  have a read at: [https://kotlinlang.org/docs/properties.html#custom-getters-and-setters](https://kotlinlang.org/docs/properties.html#custom-getters-and-setters)